app/GlobalRedux/Feautures/popup/showFavorites-slice.tsx

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { FavoritesPopupState } from '../../../../types/popup-types';
+import { PayloadAction } from '@reduxjs/toolkit';
+
+const initialState: FavoritesPopupState = {
+  showFavorites: false,
```

Comment:
ok so the same with rest of files. One file with one reducer states.

Reply:
Done